tests/test_spark.py

```diff
@@ -321,7 +322,6 @@ def test_s3_env():
 def test_foldered_spark():
     service_name = utils.FOLDERED_SPARK_SERVICE_NAME
     zk = 'spark_mesos_dispatcher__path_to_spark'
-    utils.teardown_spark(service_name=service_name, zk=zk)
```

Comment:
This is an optimisation to not have spark uninstalled and installed all the time, correct?

Reply:
not exactly, the next line after this one calls method `utils.require_spark` which includes teardown as a part of it.